orc8r/cloud/go/services/directoryd/types/types.go

```diff
@@ -77,6 +77,15 @@ func (m *DirectoryRecord) GetSessionID() (string, error) {
 	return strippedSid, nil
 }
 
+// GetLocation returns the most recent location in a directory record's
+// location history.
+func (m *DirectoryRecord) GetLocation() (string, error) {
```

Comment:
Consider `(string, bool)` return type, is easier to semantically deal with "not found" than it is to determine black-box whether an err means "not found" vs. "bad shit"